src/shop-app.html

```diff
@@ -368,7 +370,8 @@
         'clear-cart': '_onClearCart',
         'change-section': '_onChangeSection',
         'announce': '_onAnnounce',
-        'dom-change': '_domChange'
+        'dom-change': '_domChange',
+        'iron-fallback-selection-triggered': '_onFallbackSelectionTriggered'
```

Comment:
nit: the *iron* prefix gives the impression that this event is part of some iron element's API. Also, it' isn't obvious that the event is used to show a warning. Wdyt about renaming that event as `show-invalid-url-warning`? otherwise LGTM.